### PR TITLE
Ensure lowercase on user name formatting for page title profile routes

### PR DESCRIPTION
**Pull request by mcifani**

### CODE REVIEW

File: app/components/modules/Header.jsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ class Header extends React.Component {
             page_title = `Stolen Account Recovery`;
         } else if (route.page === 'UserProfile') {
             user_name = route.params[0].slice(1);
-            page_title = user_name;
+            page_title = user_name.toLowerCase();
```
Comment (roadscape) [inline app/components/modules/Header.jsx]: Is this change necessary? I believe username will always be lowercase in the route

File: app/components/modules/Header.jsx [hunk 2]
```diff
@@ -137,8 +137,13 @@ class Header extends React.Component {
             page_name = ''; //page_title = route.page.replace( /([a-z])([A-Z])/g, '$1 $2' ).toLowerCase();
         }
 
-        // Always format first letter of all titles capitalized for consistency & readability
-        page_title = page_title.charAt(0).toUpperCase() + page_title.slice(1);
+        // Format first letter of all titles and lowercase user name
+        if (route.page === 'UserProfile') {
+            page_title = page_title.charAt(0) + page_title.slice(1);
```
Comment (roadscape) [inline app/components/modules/Header.jsx]: This line can be removed, we only need to handle `route.page !== 'UserProfile'`

File: app/components/modules/Header.jsx (hunk 1 above)
Comment (mcifani) [inline app/components/modules/Header.jsx]: Yep, correct the username always is lowercase so it's not needed.
